modules/openshift-developer-cli-basic-odo-cli-commands.adoc

```diff
@@ -0,0 +1,522 @@
+  # all unattached or unmounted storage in the current application
+  odo storage list
+----
+
+== unlink
+
+.Example using unlink
+
+----
+  # Unlink the 'my-postgresql' service from the current component
+  odo unlink my-postgresql
+  
+  # Unlink the 'my-postgresql' service  from the 'nodejs' component
+  odo unlink my-postgresql --component nodejs
+  
+  # Unlink the 'backend' component from the current component (backend must have a single exposed port)
+  odo unlink backend
+  
+  # Unlink the 'backend' service  from the 'nodejs' component
+  odo unlink backend --component nodejs
+  
+  # Unlink the backend's 8080 port from the current component
+  odo unlink backend --port 8080
+----
+
+Unlink component or a service.
+
+For this command to be successful, the service or component musr have been linked prior to the invocation using `odo link`.
+
+== update
+
+Update the source code path of a component
+
+.Example using update
+
+----
+  # Change the source code path of a currently active component to local (use the current directory as a source)
+  odo update --local
+  
+  # Change the source code path of the frontend component to local with source in ./frontend directory
+  odo update frontend --local ./frontend
+  
+  # Change the source code path of a currently active component to git
+  odo update --git https://github.com/openshift/nodejs-ex.git
+  
+  # Change the source code path of the component named node-ex to git
+  odo update node-ex --git https://github.com/openshift/nodejs-ex.git
+  
+  # Change the source code path of the component named wildfly to a binary named sample.war in ./downloads directory
+  odo update wildfly --binary ./downloads/sample.war
+----
+
+== url
+
+Expose a component to the outside world. 
+
+.Example using url
+
+----
+  # Create a URL for the current component with a specific port
+  odo url create --port 8080
+  
+  # Create a URL with a specific name and port
+  odo url create example --port 8080
+  
+  # Create a URL with a specific name by automatic detection of port (only for components which expose only one service port)
+  odo url create example
+  
+  # Create a URL with a specific name and port for component frontend
+  odo url create example --port 8080 --component frontend
+
+  # Delete a URL to a component
+  odo url delete myurl
+
+  # List the available URLs
+  odo url list
+----
+
+The URLs that are generated using this command can be used to access the deployed components from outside the cluster.
+
+== utils
+
+Utilities for terminal commands and modifying odo configurations.
+
+.Example using utils
+
+----
+  # Bash terminal PS1 support
+  source <(odo utils terminal bash)
+  
+  # Zsh terminal PS1 support
+  source <(odo utils terminal zsh)
+
+----
+
+
+== version
+
+
+Print the client version information.
+
+.Example using version
+
+----
+  # Print the client version of odo
+  odo version
+----
+
+== watch
+
+Watch for changes and updates the component upon a change.
```

Comment:
Should this be: Watch for changes and updates to the component.

Reply:
@boczkowska @bergerhoffer `odo` watches for new changes and then updates the component(=applies these changes) automatically. 
I'm not sure how to reword it. Smth like "Make odo watch for changes and update the component upon the changes automatically"? Would that be clearer?